en/components/list.md

```diff
@@ -6,11 +6,14 @@ _keywords: angular list, angular component, ignite ui for angular, angular list
 
 # Angular List View
 
-<p class="highlight">The Ignite UI for Angular List component displays rows of items and supports one or more header items as well as search and filtering of list items. Each list item is completely templatable and will support any valid HTML or Angular component. </p>
+<p class="highlight">
```

Comment:
```suggestion
```

Reply:
I don't want to highlight only the first paragraph as it wont be consistent with the rest of the topics